packages/styles/scss/patterns/listsection/_listsection-item.scss

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection-item {
+    .#{$prefix}--listsection-item {
+      color: $text-01;
+      &__title {
+        @include type-style('expressive-heading-02');
+      }
+      &__content {
+        @include type-style('body-long-02');
+      }
+      &__link {
+        color: $link-01;
+        text-decoration: none;
+        outline: none;
+        display: inline-block;
+        border: solid 2px transparent;
+        @include type-style('body-short-02');
+        &:visited {
+          color: $text-02;
+        }
+        &:focus {
+          border-color: $focus;
+        }
+        &:hover {
+          text-decoration: underline;
+          cursor: pointer;
+        }
+        &__inner {
+          display: flex;
+          align-items: center;
+        }
+        &__icon {
+          fill: $link-01;
+          margin-left: $spacing-03;
+        }
+      }
```

Comment:
Why not just use carbon's `Link`, or the `LinkWithIcon` component might be more accurate?

Reply:
It was the idea to swap out with global component. I think we can use LinkWithIcon here.

Reply:
I'm not sure I'm following. Both the `LinkWithIcon`, and `Link` component allow you to include the `target` attribute.